sklearn/decomposition/_dict_learning.py

```diff
@@ -1142,6 +1142,33 @@ def transform(self, X):
         check_is_fitted(self)
         return self._transform(X, self.components_)
 
+    def _inverse_transform(self, code, dictionary):
+        """Private method allowing to accommodate both DictionaryLearning and
+        SparseCoder."""
+        if self.split_sign:
```

Comment:
@rflamary can you just see what test needs to be updated to have these lines covered? thanks

Reply:
It is done (slight modification of the test checking that `split_sign` works). Now all tests pass